### DISCUSSION TITLE
SQLSTATE[42S21]: Column already exists: 1060 Duplicate column name 'visibility' (Connection: mysql, SQL: alter table `media` add `visibility` varchar(255) not null default 'public' after `disk`)

### DISCUSSION THREAD

**Discussion by devhoussam1998** [awcodes/filament-curator#335] [Q&A] [answered]
after runing: `php artisan curator:upgrade`

<img width="959" alt="Screenshot 2023-10-30 at 20 44 55" src="https://github.com/awcodes/filament-curator/assets/121255432/148c5c69-47fb-4716-91d6-788ec819ff63">

**Comment by awcodes** [comment]
Are you upgrading from v2 to v3.?

**Reply by devhoussam1998** [reply]
yeah

**Reply by awcodes** [reply, marked as answer]
Then something else is going on here. That column doesn't exist in V2. Are you migrating for a model that uses spatie media library by any chance.?

**Reply by devhoussam1998** [reply]
I install the package again it's work now thank you
